include/net/ethernet_mgmt.h

```diff
@@ -10,7 +10,7 @@
  */
 
 #ifndef __ETHERNET_MGMT_H__
-#define __ETHERNET_MGMT_H_
+#define __ETHERNET_MGMT_H__
```

Comment:
Had to stare at that far longer than needed to figure it out.  Clang can detect typos in include guards?  Nice!